docs/maps/index.asciidoc

```diff
@@ -12,16 +12,45 @@ With *Elastic Maps*, you can:
 * Create maps with multiple layers and indices.
 * Upload GeoJSON files into Elasticsearch.
 * Embed your map in Dashboards.
-* Plot individual documents or use aggregations to plot any data set, no matter how large.
 * Create choropleth maps.
-* Use data driven styling to symbolize features from property values.
+* Use data driven styling to symbolize features from data values.
 * Focus the data you want to display with searches.
 
 Start your tour of *Elastic Maps* with the <<maps-getting-started, getting started tutorial>>.
```

Comment:
Let's figure out a for this not to get lost after the bulleted list.   What about moving it to the end and including some bold:

**Ready to get started?** Start your tour of *Elastic Maps* with the <<maps-getting-started, getting started tutorial>>.

Reply:
Maybe I should move the first image that separates the bulleted list from the header sections and move it under the `Create maps with multiple layers and indices` header?

Reply:
Yes, give it a try and see how it works.